### ISSUE TITLE
When you don't provide a CC param, interactions don't work with latest NVDA

### ISSUE THREAD

**Issue by julianna-langston** [julianna-langston/chart2music#466]
If you don't provide a CC element, then the provided chart container element receives `role="status"`. So, it doesn't receive `role="application"`, and NVDA doesn't realize that keystrokes should be passed through. While you can manually enter forms mode, it's not obvious to the user this should happen.

**Comment by julianna-langston** [comment]
Fixed in 1.17.0, which is now live.
